pycript/Reqcheck.py

```diff
@@ -110,20 +110,9 @@ def decrypt_and_update_parameters(extender, currentreq, decryptionpath, selected
                 output = extender.helpers.stringToBytes(dumps(json_object,separators=(',', ':')))
                 currentreq = extender.helpers.buildHttpMessage(header, output)
                 break
-
-        # if BOTH is selected first update GET param values, then form body
+        
         else:
-            if param.getType() == IParameter.PARAM_URL:
-                    decrypted_param =  Parameterdecrypt(selectedlang, decryptionpath, param.getValue())
-                    currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), decrypted_param, param.getType()))
-
-            elif param.getType() == IParameter.PARAM_BODY:
-                decrypted_param =  Parameterdecrypt(selectedlang, decryptionpath, param.getValue())
-                currentreq = extender.helpers.updateParameter(currentreq, extender.helpers.buildParameter(param.getName(), decrypted_param, param.getType()))
-
-    # get the updated parameters from the update request
-    parameters = extender.helpers.analyzeRequest(currentreq).getParameters()
-    header = extender.helpers.analyzeRequest(currentreq).getHeaders()
+            print("Something went wrong, Please check the request and logger")
 
     for param in parameters:
         if selected_method == "BOTH" and param.getType() == IParameter.PARAM_JSON:
```

Comment:
1. JSON has to be parsed manually there is a separate function for it. `update_raw_value` is not handling JSON objects. 

2. JSON as a whole object can be complex and all possible conditions and scenarios are checked within `update_json_value`, you are calling `update_raw_value`  for JSON which never handles JSON objects.

3. JSON is parsed as Oan bject Loop of the parameters are worthless and will not handle dict, and other combinations of JSON within loop